core/utils.py

```diff
@@ -16,4 +18,19 @@ def render(self, name, value, attrs=None, **kwargs):
                 )
             )
         output.append(super().render(name, value, attrs, **kwargs))
-        return mark_safe(u''.join(output))
\ No newline at end of file
+        return mark_safe(u''.join(output))
+
+
+def send_order_emails(user):
+    mail.send_mail(subject='Подтверждение заявки.',
+                   message=f'Здравствуйте {user.get_full_name()}.'
+                           f' Ваша заявка была принята в работу, '
+                           f'скоро с вами свяжется сотрудник нашего агентства.',
+                   from_email='antalyadom@telfia.com',
+                   recipient_list=(user.email,))
+    for admin in User.objects.filter(is_staff=True).values(
+            'first_name', 'last_name', 'email').distinct():
+        mail.send_mail(subject='Новая заявка.',
+                       message=f'Поступила заявка от {user.get_full_name()}.',
+                       from_email='antalyadom@telfia.com',
```

Comment:
Это нужно из сеттингов взять
```suggestion
                       from_email=settings.EMAIL_REPLY_TO
```